docs/upgrade/focal_migration.rst

```diff
@@ -0,0 +1,305 @@
+
+This migration will require on-premises access to the servers, and a complete
+reinstallation of Ubuntu and SecureDrop. In-place upgrades and 
+remote upgrades via SSH are not currently supported. We recommend that you 
+plan a two day maintenance window to perform and test the migration.
+
+At a high level, the migration process consists of:
+
+- Taking a backup of the current instance
+- Installing Ubuntu 20.04 LTS and SecureDrop from scratch
+- Applying the backup
+
+Instances that already have :doc:`v3 onion services <../v3_services>` enabled 
+and follow our migration guide will be able to 
+preserve their existing *Source* and *Journalist Interface* onion URLs.
+
+.. note::
+   Coordinate with journalists and sources during this migration. Your
+   instance will be offline while you perform these steps.
+
+   Coordinate with the team maintaining your Landing Page. You may wish to
+   publish a notice about a planned maintenance window. Depending on
+   :ref:`your migration scenario <choose_migration_path>`, you may also
+   need to coordinate the publication of your new *Source Interface* onion 
+   URL so that sources can reach you.
+
+.. _focal_prep:
+
+Preparation
+~~~~~~~~~~~
+Before migrating, complete the following steps:
+
+#. :ref:`Consider a hardware upgrade <consider_hardware_upgrade>`
+#. :ref:`Choose your migration path <choose_migration_path>` and plan your
+   maintenance window
+#. Coordinate with journalists to 
+   :ref:`delete old submissions from the server <prune_submissions>`             
+#. :ref:`Check your SecureDrop version (servers) <check_server_versions>`
+#. :ref:`Check your SecureDrop version (workstations) <check_workstation_versions>`
+#. :ref:`Verify SSH access <verify_ssh_access>`
+#. :ref:`Download and verify the 
+   Ubuntu 20.04 LTS (Focal) installation media <download_focal>`
+
+
+.. _consider_hardware_upgrade:
+
+Consider a hardware upgrade
+---------------------------
+If you are running hardware that is not currently listed in our
+:ref:`hardware recommendations <Specific Hardware Recommendations>`, we
+recommend that you also plan a hardware refresh as part of this migration,
+particularly if your hardware is reaching end-of-life.
+This has the following benefits:
+
+   - It ensures that all system components will continue to receive security
+     updates.
+   - It reduces the risk of hardware compatibility issues with future
+     releases of SecureDrop.
+   - It will allow you to keep your current installation online during much of
+     the two-day maintenance window.
+   - If your hardware is due for replacement anyway, combining the OS upgrade 
+     and the hardware upgrade will save you time. 
+
+If you have a support agreement with Freedom of the Press Foundation,
+please coordinate your maintenance window with us, so we can ensure that our team
+can provide support in a timely manner. In any event, please do not hesitate to
+:ref:`contact us <contact_us>` for assistance.
+
```

Comment:
Could link to [Decommission SecureDrop](https://docs.securedrop.org/en/stable/decommission.html) here, to remind them to take the proper care with the old hardware.